crates/cfsctl/src/main.rs

```diff
@@ -339,7 +339,7 @@ async fn main() -> Result<()> {
             fs.print_dumpfile()?;
         }
         Command::Mount { name, mountpoint } => {
-            repo.mount(&name, &mountpoint)?;
+            repo.mount_at(&name, &mountpoint)?;
```

Comment:
Please merge this into the commit that made this change.  Each commit ought to compile cleanly and pass tests.